indico/modules/events/timetable/util.py

```diff
@@ -332,3 +334,17 @@ def get_time_changes_notifications(changes, tzinfo, entry=None):
         if msg:
             notifications.append(msg.format(format_time(new_time, timezone=tzinfo)))
     return notifications
+
+
+@memoize_request
+def get_top_level_entries(event):
+    return event.timetable_entries.filter_by(parent_id=None).all()
+
+
+@memoize_request
+def get_nested_entries(event):
+    entries = event.timetable_entries.filter(TimetableEntry.parent_id.isnot(None)).all()
+    result = defaultdict(list)
+    for x in entries:
```

Comment:
`for entry in entries:` - I'd rather keep `x` and similar variable names inside list/dict/.. comprehensions